Initial library code

vaderkos opened the pull request.

vaderkos committed:
Initial library code

Mirosel committed:
Refactoring, simplyfication, Resolved ScopedFlare wrong type acceptance issue

vaderkos committed:
Lerna monorepo, refactoring, separating packages

vaderkos committed:
Some fixes, bugfix in isFlare function

vaderkos committed:
v1.0.1-alpha.0

vaderkos committed:
Version to 0

vaderkos committed:
Package.json update

hamboomger commented on src/lib/flare-serializable/flare-serializable.spec.ts:
```diff
@@ -0,0 +1,80 @@
+            'stack',
+            'cause'
+        )
+        expect(ser).to.not.include.all.keys(
```
It's actually described as a best practice to always prefer `to.not.include.any.keys(...)` as opposed to `.to.not.include.all.keys(..)` as the latter will fail only if all of 3 keys are present. (https://www.chaijs.com/api/bdd/#method_include)

vaderkos committed:
Initial case packages generation with gulp

vaderkos committed:
Beta

vaderkos committed:
Basic flares packages implementation

vaderkos committed:
Proper bootstrapping and linking. Some improvements. Flattened versions to 1.0.0-beta

vaderkos committed:
v1.0.0-beta.0

vaderkos committed:
Change base package.json version to 1.0.0-beta.0

vaderkos committed:
Git head added

vaderkos committed:
v1.0.0-beta.2

vaderkos committed:
v1.0.0-beta.3

vaderkos committed:
v1.0.0-beta.5

vaderkos committed:
v1.0.0-beta.6

vaderkos committed:
Add contents for lerna to published npm packages

vaderkos committed:
v1.0.0-beta.7

vaderkos committed:
v1.0.0-beta.8

vaderkos committed:
v1.0.0-beta.9

vaderkos committed:
v1.0.0-beta.10

vaderkos committed:
v1.0.0-beta.11

vaderkos committed:
v1.0.0-beta.12

vaderkos committed:
v1.0.0-beta.12

vaderkos committed:
v1.0.0-beta.13

vaderkos committed:
v1.0.0-beta.13

vaderkos committed:
v1.0.0-beta.17

vaderkos committed:
v1.0.0-beta.18

vaderkos committed:
Created @flares/express. Various fixes

vaderkos committed:
v1.0.0-beta.19

vaderkos committed:
Added license files

vaderkos committed:
Added readme file for @flares/express

vaderkos committed:
v1.0.0-beta.20

vaderkos committed:
Test commit dists for lerna proper deploy

vaderkos committed:
v1.0.0-beta.21

vaderkos committed:
dists removed

vaderkos committed:
v1.0.0-beta.25

vaderkos committed:
v1.0.0-beta.27

vaderkos committed:
Fixed imports on done packages. Express integration improved

vaderkos committed:
v1.0.0-beta.28

vaderkos committed:
Improved keywords

dependabot[bot] committed:
Bump ini from 1.3.5 to 1.3.8

Bumps [ini](https://github.com/isaacs/ini) from 1.3.5 to 1.3.8.
- [Release notes](https://github.com/isaacs/ini/releases)
- [Commits](https://github.com/isaacs/ini/compare/v1.3.5...v1.3.8)

Signed-off-by: dependabot[bot] <support@github.com>

vaderkos committed:
v1.0.0-beta.29

vaderkos committed:
Merge pull request #5 from vaderkos/dependabot/npm_and_yarn/ini-1.3.8

Bump ini from 1.3.5 to 1.3.8

vaderkos committed:
Test readme.
More methods.
Improved @flare/express (new features).
Refactoring

vaderkos committed:
Merge branch 'develop' of https://github.com/vaderkos/flares into develop

vaderkos committed:
v1.0.0-beta.30

vaderkos committed:
Changes in docs of @flares/express

vaderkos committed:
Merge branch 'develop' of https://github.com/vaderkos/flares into develop

vaderkos committed:
Testing readme.
More methods.
Improved SerializableFlare function.
More test for Flare and SerializableFlare.

vaderkos committed:
Removed SerializableFlare from default response ar @flare/express

vaderkos committed:
ThrowErroRequestHandler

vaderkos committed:
Tryed in readme for @flares/express

vaderkos committed:
Another fixes in readme for @flares/express

vaderkos committed:
v1.0.0-beta.31

vaderkos committed:
v1.0.0-beta.32

hamboomger commented on packages/core/src/flare/flare.spec.ts:
```diff
@@ -0,0 +1,230 @@
+    })
+
+    describe('Flare static methods', () => {
+        const decline = [undefined, 99999, Symbol('test'), class T {}, function T () {}, () => undefined]
```
Nice

hamboomger commented on packages/express/README.md:
````diff
@@ -0,0 +1,47 @@
+* [```@flares/pascal-case-flares```](https://www.npmjs.com/package/@flares/pascal-case-flares)
+* [```@flares/pascal-case-code-flares```](https://www.npmjs.com/package/@flares/pascal-case-code-flares)
+
+## Basic Usage
````
Noice

hamboomger commented on packages/camel-case-code-flares/tsconfig.json:
```diff
@@ -0,0 +1,13 @@
+  "include": [
+    "./src/**/*"
+  ],
+  "exclude": [
+    "./node_modules",
+    "./dist"
+  ]
```
Redundant statement, since paths below are not included in the first place
